discord/client.py

```diff
@@ -1990,7 +1990,7 @@ async def update_role_connection_metadata_records(
 
         Parameters
         ----------
-        *role_connection_metadata: :class:`ApplicationRoleConnectionMetadata`
+        *role_connection_metadata: List[:class:`ApplicationRoleConnectionMetadata`]
```

Comment:
I believe this was supposed to be no list because of how typehint s are handled be python

Reply:
Indeed, expanding args is not considered a list.

Reply:
yeah, there was some talk about it about typing *args, **kwargs in python but overall it wasn't introduced as far as I can see and I thought a bit about other programming languages - it will be reverted in next commit